### PR TITLE
[ci] Run all python files in the *_source folders

### PR DESCRIPTION
**Pull request by clee2000**
Run all python files in the *_source folders.

Make build.sh slightly less verbose

### CODE REVIEW

File: .jenkins/build.sh [hunk 1]
```diff
@@ -66,42 +66,50 @@ if [[ "${JOB_BASE_NAME}" == *worker_* ]]; then
 
   # Step 4: If any of the generated files are not related the tutorial files we want to run,
   # then we remove them
+  set +x
   for filename in $(find docs/beginner docs/intermediate docs/advanced docs/recipes docs/prototype -name '*.html'); do
     file_basename=$(basename $filename .html)
     if [[ ! " ${FILES_TO_RUN} " =~ " ${file_basename} " ]]; then
+      echo "removing $filename"
```
Comment (ZainRizvi) [inline .jenkins/build.sh:73]: fyi, bash scripts can have functions too :)

https://linuxize.com/post/bash-functions/

File: .jenkins/validate_tutorials_built.py [hunk 2]
```diff
@@ -5,13 +5,15 @@
 
 REPO_ROOT = Path(__file__).parent.parent
 
-# files that are ok to have 0 min 0 sec time, probably because they don't have any python code
-OK_TO_NOT_RUN = [
-    "beginner/basics/intro.html",  # no code
-]
+# files not ending in "tutorial.py" are not run by sphinx (see sphinx_gallery_conf in conf.py),
```
Comment (ZainRizvi) [inline .jenkins/validate_tutorials_built.py:8]: Does this mean we don't actually have any tutorials we currently know to be broken?

**Review by ZainRizvi — APPROVED** [review]

File: .jenkins/validate_tutorials_built.py (hunk 2 above)
Comment (clee2000) [reply .jenkins/validate_tutorials_built.py:8]: yes, or at least i am not aware of them

**Comment by malfet** [comment]
Hmm, wouldn't those renames break consistency?

**Review by malfet — APPROVED** [review]
LGTM

**Pull request merged**